packages/flutter/lib/src/loader/loader_app.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:flutter/material.dart';
+
+void main() {
+  runApp(new _LoaderScreen());
```

Comment:
since it's stateless now you can just put the whole thing in void main()'s runApp() call.